### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by malberda**
### Why:

Fixes small errors or inconsistencies

### What's being changed:

Changed a few small text lines to more clearly explain the steps

### Check off the following:

- [ X] For workflow changes, I have verified the Actions workflows function as expected.
- [ X] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ If the checks don't appear or if the checks are stuck in progress, there's a few
    ```markdown
    _underscore_
    ```
+1. Replace the * with an _ in line 115 of README.md
```
Comment (markpatterson27) [inline README.md:131]: Again, I don't think this is the way to go. I think the activity works better by getting the learner to use the Actions log to identify what and where the error is. Maybe a _hints_ detail block could be used to help stuck learners.

The preceding sentence has a typo in it which makes the activity very confusing. This definitately needs to be fixed. Perhaps something like:

1. Update the code in the `ci` branch to get the test to pass. You need to change the incorrect code to look something like this:
   ```markdown
   _underscore_
   ```
   <details><summary>hint</summary>
   You need to change line 115 of <code>README.md</code> to use _ for emphasis insead of *.
   </details>

**Review by markpatterson27 — COMMENTED** [review]
A number of people have reported finding it difficult to understand what the activity in step 2 is asking. See:
- #10 
- [Where is the _underscore_?](https://github.com/orgs/skills/discussions/77)

I definitely think the explanation in the activity needs improved, but I don't think the suggested changes are the way to go. I have added my own suggests in the review.

File: README.md [hunk 2]
```diff
@@ -76,12 +76,12 @@ First, let's add a workflow to lint our Markdown files in this repository.
...
-        run: |
-          npm install remark-cli remark-preset-lint-consistent
-          npx remark . --use remark-preset-lint-consistent --frail
+        - name: Run markdown lint
+          run: |
+            npm install remark-cli remark-preset-lint-consistent
+            npx remark . --use remark-preset-lint-consistent --frail
    ```
-   > We expect this to create a error build. We'll fix this in the next step.
+   > Even after the code is indented properly in 'ci.yml', you will see a build error in GitHub Actions. We'll fix this in the next step.
```
Comment (heiskr) [inline README.md:84]:
```suggestion
   > Even after the code is indented properly in `ci.yml`, you will see a build error in GitHub Actions. We'll fix this in the next step.
```

File: README.md (hunk 1 above)
Comment (heiskr) [inline README.md:131]:
```suggestion
```

Comment (heiskr) [reply README.md]: > I think the activity works better by getting the learner to use the Actions log to identify what and where the error is

That's a major learning objective for this course for sure. We moved this to another, shorter file to make it easier to find.

Comment (sinsukehlab) [reply README.md]: Step 2 in README.md
> _By looking at the logs, can you identify which tests failed?_ To find it, go to one of the failed builds and scrolling through the log. Look for a section that lists all the unit tests. We're looking for the name of the test with an "x".

What do you mean by 'the name of the test with an "x"'?

**Pull request merged**